src/main/kotlin/graphics/scenery/DefaultNode.kt

```diff
@@ -48,7 +48,6 @@ open class DefaultNode(name: String = "Node") : Node, Networkable {
         return true
     }
 
-    override var nodeType = "Node"
```

Comment:
Will this cause any trouble in other parts of scenery that rely on this node type?